lib/go/test/subedition_test.go

```diff
@@ -0,0 +1,749 @@
+package test
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/onflow/cadence"
+	jsoncdc "github.com/onflow/cadence/encoding/json"
+
+	"github.com/dapperlabs/nba-smart-contracts/lib/go/contracts"
+	"github.com/dapperlabs/nba-smart-contracts/lib/go/templates"
+
+	"github.com/onflow/flow-go-sdk/crypto"
+	sdktemplates "github.com/onflow/flow-go-sdk/templates"
+	"github.com/onflow/flow-go-sdk/test"
+
+	"github.com/onflow/flow-go-sdk"
+	"github.com/stretchr/testify/assert"
+)
+
+// This test tests the pure functionality of the smart contract
+func TestSubeditions(t *testing.T) {
+	b := newBlockchain()
```

Comment:
Recently added a few helpers that should make this a bit less verbose to setup - take a look at NewTopShotTestBlockchain